components/catalogs/back/src/Application/Persistence/UpdateCatalogProductValueFiltersQueryInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Catalogs\Application\Persistence;
+
+/**
+ * @author    Willy Mesnage <willy.mesnage@akeneo.com>
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface UpdateCatalogProductValueFiltersQueryInterface
+{
+    /**
+     * @param array{channel?: array<string>} $productValueFilters
```

Comment:
```suggestion
     * @param array{channels?: array<string>} $productValueFilters
```

It's `channels`, not `channel`.

https://github.com/akeneo/pim-community-dev/blob/master/components/catalogs/back/src/Infrastructure/Persistence/GetProductsQuery.php#L72

Reply:
Ah! I need to rename it everywhere! Thanks marketing...